addon/components/sortable-group.js

```diff
@@ -29,21 +38,40 @@ export default Component.extend({
             });
         });
     },
-    init() {
+    async initializeGroup() {
         //Default sortable group array to be null, these will be added when the sortable groups insert into the DOM
         if (!get(this, 'fastboot.isFastBoot')) {
+            // See https://github.com/ef4/ember-auto-import/issues/98
+            const { Sortable, Plugins } = await import('@shopify/draggable');
+            const plugins = A();
+            if (get(this, 'resizeMirror')) {
+                plugins.pushObject(Plugins.ResizeMirror);
+            }
             const mirror = {
                 constrainDimensions: get(this, 'constrainDimensions')
             }
             const sortable = new Sortable([], {
                 draggable: '.sortable-item',
                 mirror,
-                plugins: [Plugins.ResizeMirror]
+                plugins,
+            });
+            setProperties(this, {
+                sortable,
+                plugins: Plugins
             });
-            set(this, 'sortable', sortable);
+            //Public Events
             this.initializeEventListeners();
+            //Private Events
+            get(this, '_events').forEach(eventName => {
+                sortable.on(eventName, (event) => {
+                    this.trigger(eventName, event);
+                });
+            });
+            this.trigger('setupContainers');
         }
-
+    },
+    init() {
+        this.initializeGroup();
```

Comment:
is it possible to call this from `didInsertElement` instead? If you could, then you wouldn't need the isFastBoot check or the `fastboot` computed prop.